charts/spire/charts/spire-agent/templates/configmap.yaml

```diff
@@ -12,6 +12,11 @@ agent:
   trust_bundle_path: "/run/spire/bundle/bundle.crt"
   {{- end }}
   trust_domain: {{ include "spire-lib.trust-domain" . | quote }}
+  sds:
+    default_svid_name: {{ .Values.sds.defaultSvidName | quote }}
+    default_bundle_name: {{ .Values.sds.defaultBundleName | quote }}
+    default_all_bundles_name: {{ .Values.sds.defaultAllBundlesName | quote }}
+    disable_spiffe_cert_validation: {{ .Values.sds.disableSpiffeCertValidation }}
```

Comment:
Below you are stating this is an optional configuration, however here in the implementation it isn't.

Maybe we should add a enabled flag to conditionally enable or disable these settings.